addons/binding/org.openhab.binding.yamahareceiver/README.md

```diff
@@ -33,13 +33,15 @@ Zone control channels are:
 
 Playback control channels are:
 
-* `preset#playback_channels`: Set a preset.
+* `preset#playback_channels`: Set a preset. Not supported by Spotify.
 * `playback#playback_channels`: Set a play mode or get the current play mode.
 * `playback_station#playback_channels`: Get the current played station (radio).
 * `playback_artist#playback_channels`: Get the current played artist.
 * `playback_album#playback_channels`: Get the current played album.
 * `playback_song#playback_channels`: Get the current played song.
-            
+* `playback_song_image_url#playback_channels`: Get the current played song image URL (currently Spotify input only). OpenHAB Type `String`.
```

Comment:
I thought it is OpenHab? Would be consistent with Arp instead of ARP and Icmp instead of ICMP (that's where you corrected me for instance, although uppercase would be correct).